### PR TITLE
Create get-started-vwrs.md

### PR DESCRIPTION
**Pull request by diana-macrometa**
Get started with VWRs topic.

@rahul-naik1 and @joubert-macrometa -  I need to know more about the workflow here. Am I anywhere close to correct? What does the user need to do or know before they even try to set up the waiting room?

### CODE REVIEW

File: docs/photoniq/vwrs/configure-vwrs-edgeworker.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ sidebar_position: 30
 title: Configure a VWRs EdgeWorker
 ---
 
-Import the `VirtualWaitingRoom` class from the virtualWaitingRoom.js file to configure a waiting room.
+Import the `VirtualWaitingRoom` class from the [virtualWaitingRoom.js file](NEED FILE) to configure a waiting room.
```
Comment (diana-macrometa) [inline docs/photoniq/vwrs/configure-vwrs-edgeworker.md]: @rahul-naik1 and @joubert-macrometa, what file is this referring to? Also, where does the user do this? Is there an Akamai page we can link to or something? Or should we say "Have your Akamai SE set this up for you" or something similar?

Comment (joubert-macrometa) [reply docs/photoniq/vwrs/configure-vwrs-edgeworker.md]: In the EdgeWorker, import the 'VirtualWaitingRoom' class from the VWRoom Client[https://www.npmjs.com/package/vwroomclient.] library.  You can get this package from Node Package Manager (NPM).  This integrates the VWRs with the EdgeWorker.

@diana-macrometa you will have to clean this up, feels a little clunky.

Comment (diana-macrometa) [reply docs/photoniq/vwrs/configure-vwrs-edgeworker.md]: Is XXX the actual name?

**Review by joubert-macrometa — APPROVED** [review]

File: docs/photoniq/vwrs/configure-vwrs-edgeworker.md (hunk 1 above)
Comment (diana-macrometa) [reply docs/photoniq/vwrs/configure-vwrs-edgeworker.md]: Thanks! This is now just waiting on the updated API.

File: docs/photoniq/vwrs/configure-vwrs-edgeworker.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ sidebar_position: 30
 title: Configure a VWRs EdgeWorker
 ---
 
-Import the `VirtualWaitingRoom` class from the virtualWaitingRoom.js file to configure a waiting room.
+In the EdgeWorker, import the `VirtualWaitingRoom` class from the `xxx` Node Package Manager (NPM). This integrates the VWRs with the EdgeWorker.
```
Comment (rahul-naik1) [inline docs/photoniq/vwrs/configure-vwrs-edgeworker.md]: Looks like @joubert comment is lost. XXX is for vwroomclient with link to https://www.npmjs.com/package/vwroomclient

Comment (diana-macrometa) [reply docs/photoniq/vwrs/configure-vwrs-edgeworker.md]: Thanks! Updated.

**Pull request merged**